File: bot.py
from telegram import Update, InlineKeyboardButton, InlineKeyboardMarkup, WebAppInfo
from telegram.ext import Application, CommandHandler, ContextTypes

TOKEN = "8690035396:AAFfRrjROUl8L6OkQoG6qhqYUYo-IXT3nvY"
WEBAPP_URL = "https://footy-tap.vercel.app"

async def start(update: Update, context: ContextTypes.DEFAULT_TYPE):
    keyboard = [
        [InlineKeyboardButton("🚀 LAUNCH FOOTY TAP", web_app=WebAppInfo(url=WEBAPP_URL))]
    ]

    await update.message.reply_text(
        "⚽️ Footy Tap готов!",
        reply_markup=InlineKeyboardMarkup(keyboard)
    )

app = Application.builder().token(TOKEN).build()
app.add_handler(CommandHandler("start", start))

print("Bot running...")
app.run_polling()

File: bot.js
const TelegramBot = require('node-telegram-bot-api');

const token = '8690035396:AAFfRrjROUl8L6OkQoG6qhqYUYo-IXT3nvY';

const bot = new TelegramBot(token, { polling: true });

bot.onText(/\/start/, (msg) => {
    bot.sendMessage(msg.chat.id,
        "⚽️ Footy Coin готов!\nНажми LAUNCH чтобы начать игру:",
        {
            reply_markup: {
                inline_keyboard: [
                    [
                        {
                            text: "LAUNCH ⚽️",
                            web_app: {
                                url: "https://footyopkiijk.vercel.app/"
                            }
                        }
                    ]
                ]
            }
        }
    );
});

console.log("Bot started...");
